src/main/webapp/app/components/tokenInputGroups/TokenInputGroups.tsx

```diff
@@ -9,8 +9,10 @@ import classnames from 'classnames';
 import { observable, action } from 'mobx';
 import { observer } from 'mobx-react';
 import autobind from 'autobind-decorator';
+import TokenExtendButton from './TokenExtendButton';
 
 type TokenInputGroupsProps = {
+  extend: boolean;
```

Comment:
This is definitely not a good name. The purpose of this calendar is to change token expiration date. so maybe `changeTokenExpirationDate`?